webapp/channels/src/components/widgets/menu/menu_items/menu_cloud_trial.test.tsx

```diff
@@ -205,7 +205,6 @@ describe('components/widgets/menu/menu_items/menu_cloud_trial', () => {
         };
         const store = mockStore(state);
         const wrapper = mountWithIntl(<Provider store={store}><MenuCloudTrial id='menuCloudTrial'/></Provider>);
-        console.log(wrapper.debug());
```

Comment:
not related to the change, just saw the log while running the test, then find it and deleted it.